lib/segment/src/vector_storage/dense/simple_dense_vector_storage.rs

```diff
@@ -222,6 +222,13 @@ impl<T: PrimitiveVectorElement> VectorStorage for SimpleDenseVectorStorage<T> {
         CowVector::from(T::slice_to_float_cow(self.vectors.get(key).into()))
     }
 
+    /// Get vector by key, if it exists.
+    fn get_vector_opt(&self, key: PointOffsetType) -> Option<CowVector> {
```

Comment:
Do we need something the same for multivectors?

Reply:
Good point, I will add it :+1: